spec/services/banking/bank_transactions_trimmer_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Banking::BankTransactionsTrimmer do
   describe ".call" do
-    subject { described_class.call(application) }
+    subject(:bank_transaction_trimmer) { described_class.call(application) }
```

Comment:
nit, could just use the described (class) method, but either way its an improvement